.eslintignore

```diff
@@ -1,7 +1,7 @@
 addon/StudyUtils.jsm
+addon/PioneerUtils.jsm
```

Comment:
It will be added by anyone writing a Pioneer shield study, and they should still be able to use this template (me included :) )

Reply:
Maybe just add a comment to explain what it's about? Something like:
`# This file is included for Pioneer Shield studies only; eventually it will be merged with StudyUtils.jsm`